### ISSUE TITLE
PrimitiveMemberEditors in World only parse OS locale but revert content to invariant culture on EditingFinished (decimal separators)

### ISSUE THREAD

**Issue by mpmxyz** [Yellow-Dog-Man/Resonite-Issues#1655]
### Describe the bug?

I've boiled it down to the title.

### To Reproduce

Set OS to German.
Set Resonite to English.
(Maybe) Restart Resonite. (My settings already were like that.)
Spawn Float input nodes, open inspectors to seek float fields etc.
Try to enter numbers with fractions.
Try . and , as decimal separators.

### Expected behavior

PrimitiveMemberEditors should only use the Resonite locale or invariant culture. Parsing may be lenient to support both.

### Screenshots

When not editing:
![Non-Editing](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/8538547/06ec5ae5-08c6-4b79-a962-de86b0b339ef)
When editing:
![Editing1](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/8538547/2f3dfc48-9b20-4859-86f9-070bcba0baba)
![Editing2](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/8538547/cb1e3a79-f1a0-4e20-8a9e-d68c8bedb147)

### Resonite Version Number

2024.4.9.1426

### What Platforms does this occur on?

Windows

### What headset if any do you use?

Desktop

### Log Files

[ANONYMOUS - 2024.4.9.1426 - 2024-04-11 00_13_00.log](https://github.com/Yellow-Dog-Man/Resonite-Issues/files/14938387/ANONYMOUS.-.2024.4.9.1426.-.2024-04-11.00_13_00.log)

**Comment by Frooxius** [comment]
Duplicate of: https://github.com/Yellow-Dog-Man/Resonite-Issues/issues/905

This will be fixed once Setting rework is merged in.

**Comment by mpmxyz** [comment]
Are your changes already integrated into the current pre-release build?
If yes: The things I mentioned here are actually new bugs that are not related to #905 which is "just" an inconsistency of the output nodes in relation to the inputs.
Here we've got an inconsistency _within_ number inputs.

**Comment by Banane9** [comment]
Yea, this is basically the opposite of my issue and definitely new with the pre-release 👀

**Comment by Frooxius** [comment]
I see. I didn't realize this was feedback for the prerelease, since it was not tagged as such.

I'll have a look, thank you!

**Comment by Frooxius** [comment]
@mpmxyz Can you verify if the latest pre-release fixes this for you please? I don't have German OS locale, so I can't test that specifically.

**Comment by Banane9** [comment]
This can be closed now, I think
